cnf-certification-test/manageability/suite.go

```diff
@@ -76,18 +80,17 @@ func testContainerPortNameFormat(env *provider.TestEnvironment) {
 	for _, newProtocol := range env.ValidProtocolNames {
 		allowedProtocolNames[newProtocol] = true
 	}
-	badContainers := []string{}
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
 	for _, cut := range env.Containers {
 		for _, port := range cut.Ports {
 			if !containerPortNameFormatCheck(port.Name) {
-				badContainers = append(badContainers, cut.String())
 				tnf.ClaimFilePrintf("%s: ContainerPort %s does not follow the partner naming conventions", cut, port.Name)
+				nonCompliantObjects = append(nonCompliantObjects, testhelper.NewContainerReportObject(cut.Namespace, cut.Podname, cut.Name, fmt.Sprintf("ContainerPort %s does not follow the partner naming conventions", port.Name), false))
```

Comment:
optional: consider re-using port/protocol fields defined in https://github.com/test-network-function/cnf-certification-test/pull/1207 and add 2 level report, one per container per port and one per pod

Reply:
I'll adjust this once the #1207 gets reviewed and merged.